Makefile

```diff
@@ -5,36 +5,29 @@ all:
 	@echo make restart
 	@echo make keyboard-dev
 
-install-audio-receiver:
-	wget -q https://github.com/nicokaiser/rpi-audio-receiver/archive/main.zip
-	unzip main.zip
-	rm main.zip
-	
-	cd rpi-audio-receiver-main;	sudo ./install.sh
-
-dependencies: install-audio-receiver
+dependencies:
 	sudo apt install curl virtualenv espeak mpc mpd
 	virtualenv -p /usr/bin/python3 virtualenv
 	./virtualenv/bin/pip install --upgrade -r requirements.txt
 	echo "Reboot is recommended now"
 
 install:
-	# Install the systemd service...
-	# Create systemd directories
-	mkdir -p ~/.config/systemd/user
-
-	# Copy the service file to systemd directories
-	## cp spotify-alarm-clock.service ~/.config/systemd/user
-	# Reload the systemd daemon: https://askubuntu.com/a/1143989/413683
-	## systemctl --user daemon-reload
-	# Enable the service to start system boot (I hope)
-	## systemctl --user enable spotify-alarm-clock
-
-	# copy the .service file to global systemd directory
-	cp spotify-alarm-clock.service /etc/systemd/service
+	# Installs the systemd service
+	# For the program to run in the background and start on boot
+	sudo cp spotify-alarm-clock.service /etc/systemd/system
 	sudo systemctl daemon-reload
 	sudo systemctl enable spotify-alarm-clock
```

Comment:
1. Cool, systemwide works so I removed the commented-out older version :) nice that it's less confusing now that all services are systemwide
2. This is less visible because of the other changes, but there was a bug preventing this from working in your version (`/etc/systemd/service` instead of `/etc/systemd/system`). At least that's how it behaved for me, maybe worth another test